docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -441,7 +441,7 @@ data visualizations, in about 5 minutes.
 In this section, you learn how to run the `system` module to collect metrics
 from the operating system and services running on your server. The system module
 collects system-level metrics, such as CPU usage, memory, file system, disk IO,
-and network IO statistics, as well as top-like statistics for every process
+and network IO statistics, as well as top-line statistics for every process
```

Comment:
My first impression was that this was referring to the "top" command, @dedemorton would it be accurate to change to something like this?:
```suggestion
and network IO statistics, as well as top-level statistics for every process
```

Reply:
No, it is referring to statistics similar to what you get with the top command. If you think it's too confusing, I would just delete "top-like" from the sentence.